### PR TITLE
refactor: use one build per tests

### PR DESCRIPTION
**Pull request by pyphilia**
ref #1516

### CODE REVIEW

File: src/services/item/plugins/importExport/test/index.test.ts [hunk 1]
```diff
@@ -3,12 +3,18 @@ import fs from 'fs';
 import { StatusCodes } from 'http-status-codes';
 import fetch from 'node-fetch';
 import path from 'path';
+import { ILike } from 'typeorm';
```
Comment (Defferrard) [inline src/services/item/plugins/importExport/test/index.test.ts:6]: Unused import

Comment (Defferrard) [inline src/services/item/plugins/importExport/test/index.test.ts:6]: Unused import

**Review by Defferrard — APPROVED** [review]
LGTM!

File: src/services/item/plugins/importExport/test/index.test.ts
Comment (spaenleh) [inline src/services/item/plugins/importExport/test/index.test.ts:1]: I see that you have changed the behaviour of the test to import inside a folder. Does this mean we do not test importing at the root anymore ? we could check that the import is done at the root with a query on the accessible for the user.

**Review by spaenleh — APPROVED** [review]

**Review by ReidyT — APPROVED** [review]
LGTM

**Pull request merged**